openwisp_firmware_upgrader/admin.py

```diff
@@ -200,22 +204,65 @@ def __get_rate(self, value):
     aborted_rate.short_description = _('abortion rate')
 
 
+class DeviceFirmwareForm(forms.ModelForm):
+    class Meta:
+        model = load_model('DeviceFirmware')
+        fields = '__all__'
+
+    def _get_compatible_boards(self, instance):
```

Comment:
I think this is not working, I see all the images, of different organizations, are there tests for this?